spec/round_spec.rb

```diff
@@ -0,0 +1,129 @@
+require './lib/card.rb'
+require './lib/turn.rb'
+require './lib/deck.rb'
+require './lib/round.rb'
+require './lib/card_generator.rb'
+
+RSpec.describe Round do
+  before :each do
+    cardset = CardGenerator.new('cards.txt').temp_deck
+    @deck = Deck.new(cardset)
+    @card = @deck.card_at(0)
+    @guess_wrong = "arbitrarily wrong"
+    @guess_correct = @card.answer
+    @turn_correct = Turn.new(@card.answer, @card)
+    @turn_wrong = Turn.new(@guess_wrong, @card)
+    @round = Round.new(@deck)
+  end
+
+  it 'exists' do
+    expect(@round).to be_instance_of(Round)
+  end
+
+
+  it 'correctly makes a deck' do
+    expect(@round.deck).to be_instance_of(Deck)
+    expect(@round.deck).to eq(@deck)
+  end
+
+
+  it 'has empty turn array at first' do
+    expect(@round.turns).to eq([])
+  end
+
+
+  it 'gives current card' do
+    expect(@round.current_card).to eq(@deck.card_at(@round.card_index))
+  end
+
+  it 'correctly takes a turn & stores argument' do
+    new_turn = @round.take_turn("Juneau")
+
+    expect(new_turn).to be_instance_of(Turn)
+    expect(new_turn.guess).to eq("Juneau")
+    expect(new_turn.card).to eq(@card)
+  end
+  
+
+  it 'identifies correct and incorrect guesses' do
```

Comment:
These looping constructs in this test and in other tests up to this point isn't necessary and you should be limiting your tests to method calls on the object.